util/migdb-add-unit

```diff
@@ -0,0 +1,245 @@
+#!/bin/bash
+
+SCRIPT_PATH=$(dirname "${0}")
+# shellcheck source=/home/thomas/develop/balena.io/migrate/util/migdb-common
```

Comment:
I would modify shellcheck override out, as it's incorrect on other people's machine. Instead, likely use the relative path version of this from https://github.com/koalaman/shellcheck/wiki/Directive#source if works, ie.:

```
# shellcheck source=migdb-common
```
though haven't tried it yet.

Reply:
Thanks, got it sorted properly now